### ISSUE TITLE
networking RFE - use better names for created ifcfg files that UUID

### ISSUE THREAD

**Issue by jscotka** [cockpit-project/cockpit#1161]
Hi,
I've tested creating VLAN via cockpit, and it is okay, but generated ifcfg file have UUID names, what are not easy to, like:
/etc/sysconfig/network-scripts/ifcfg-18f3ec29-8e7c-412a-a746-5cfd83bf929f 
with generated line:
NAME=18f3ec29-8e7c-412a-a746-5cfd83bf929f

there woul be also text fied for changing name for NM: NAME= in ifcfg file. then file could be named via this name (if it is possible)
     Thanks&Regards
     Honza

**Comment by stefwalter** [comment]
I think this is a NetworkManager bug.

**Comment by jscotka** [comment]
Yes. It is NM related. It is possible that there could be used some option to override using Names instead of UUIDs for file names.
But anyway it is cosmetic issue not functional.
So this issue could be closed.

**Comment by mvollmer** [comment]
The names are chosen by Cockpit, for example here:

https://github.com/cockpit-project/cockpit/blob/master/pkg/networkmanager/interfaces.js#L1494
```
           {
                connection: {
                    id: uuid,
                    autoconnect: false,
                    type: "bond",
                    uuid: uuid,
                    interface_name: iface
                },
                bond: {
                    options: {
                        mode: "balance-rr"
                    },
                    interface_name: iface
                }
            }
```

We might get away with using `iface` instead of `uuid` for `id` in this case.

**Comment by dperpeet** [comment]
fixed in #4889
